<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ayaansh" failures="2" tests="3" name="TestRunner.Runner" time="27.213" errors="0" timestamp="2024-04-15T10:48:46 IST" skipped="0">
  <testcase classname="TestRunner.Runner" name="runScenario" time="10.062">
    <failure message="confirmed expected [Administration] but found [Dashboard]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: confirmed expected [Administration] but found [Dashboard]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at POM.pomodel.verify(pomodel.java:39)
at Step.stepFile.verify_login_as(stepFile.java:92)
at ✽.verify  as "Administration"(file:///C:/Users/abhil/git/TestDemo/TestDemo/src/test/resources/rrr.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="TestRunner.Runner" name="runScenario" time="7.490"/>
  <testcase classname="TestRunner.Runner" name="runScenario" time="9.661">
    <failure message="confirmed expected [Administration!] but found [Administration]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: confirmed expected [Administration!] but found [Administration]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at POM.pomodel.verify(pomodel.java:39)
at Step.stepFile.verify_login_as(stepFile.java:92)
at ✽.verify  as "Administration!"(file:///C:/Users/abhil/git/TestDemo/TestDemo/src/test/resources/rrr.feature:30)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- TestRunner.Runner -->
